### ISSUE TITLE
Auto-translations of external PDFs

### ISSUE THREAD

**Issue by skiasaurus** [skiasaurus/tau_manifesto#19]
Google Translate's ability to handle PDF files is more limited than that for HTML files (see heading "Translate documents" [here](https://support.google.com/translate/answer/2534559)). In particular, the URL of the output is always the same, regardless of content: https://translate.googleusercontent.com/translate_f.

This isn't urgent, but we should look into options for how we might be able to apply the GT translation algorithm to PDFs that are linked to directly, like (for instance) Peter Harremoës' [paper on Al-Kashi's constant](http://www.harremoes.dk/Peter/Undervis/Turnpage/Turnpage1.pdf).

**Comment by draialexis** [comment]
The quick and dirty solution I have in mind is "hardcoding" the French version, if we can't have the URL launch a GT job dynamically.

That may create new issues though. Maybe Man was never meant to auto-translate a pdf in the first place 🧐

**Comment by skiasaurus** [comment]
> The quick and dirty solution I have in mind is "hardcoding" the French version

What does that mean exactly? And what would it entail?

> That may create new issues though. Maybe Man was never meant to auto-translate a pdf in the first place

:raised_eyebrow: (:arrow_left: that's a raised eyebrow, in case you can't see it clearly)

**Comment by draialexis** [comment]
I meant putting the pdf through some auto-translator ourselves, and adding that French version to the publication.

By "new issues", I meant that we would be adding a whole new document, rather than just providing a link. I don't know if Michael would be ok with that, and I have no idea how complicated SoftCover would make it.

**Comment by skiasaurus** [comment]
Yeah, that's way too much trouble. I think we may once again just have to rely on readers to auto-translate it for themselves, at least for now.
